lib/Kraken.php

```diff
@@ -85,7 +85,7 @@ private function request($data, $url, $type) {
         curl_setopt($curl, CURLOPT_POSTFIELDS, $data);
         curl_setopt($curl, CURLOPT_FAILONERROR, 0);
         curl_setopt($curl, CURLOPT_CAINFO, __DIR__ . "/cacert.pem");
-        curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, 1);
+        curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, 0);
```

Comment:
This would effectively render SSL useless. The proper solution here would be to remove the cacert file and have curl operate based on your operating systems certificate store, e.g. ca-certificates on Debian based systems.